experiences/branded-content/branded-content-card.test.js

```diff
@@ -1,115 +1,47 @@
-			config.cards.BrandedContentFlowcard.snapPoints.active);
-
-		expect(isAtActiveSnapPoint).equal(true);
-	});
-
-	it('close card pressing close button', async()=>{
-		await scrollCard(browser, config.cards.BrandedContentFlowcard.cardSelector, 400);
-
-		await closeCard(browser);
-
-		const isAtInitialSnapPoint = await isAtSnapPoint(browser,
-			config.cards.BrandedContentFlowcard.cardSelector,
-			config.cards.BrandedContentFlowcard.snapPoints.initial,
-			async()=>{
-				await closeCard(browser);
+const experience = require('./branded-content-card.json');
+const { test, doers, checkers } = require('../../e2e/tests');
+
+const customCheckers = {
+	croupierMatches: () => {
+		return async () => {
+			const dynamic_flowcards = JSON.stringify({
+				name: 'BrandedContentFlowcard',
+				relevance: 100,
+				url: 'https://playground.marfeel.com/experiences/branded-content/branded-content-card.html',
+				renderProps: JSON.stringify(experience.cards.BrandedContentFlowcard)
 			});
 
-		expect(isAtInitialSnapPoint).equal(true);
-	});
-
-	it('minimize card dragging it down when status is "initial"', async() => {
-		if (!config.cards.BrandedContentFlowcard.features.removable) {
-			// Restore initial status after infinite scroll
-			await scrollBy(browser, -1800);
-			await minimizeCard(browser, config.cards.BrandedContentFlowcard.cardSelector);
-
-			const isAtMinimizedSnapPoint = await isAtSnapPoint(
-				browser,
-				config.cards.BrandedContentFlowcard.cardSelector,
-				config.cards.BrandedContentFlowcard.snapPoints.minimised
-			);
+			const croupier =  await getFlowcardJson({}, `https://flowcards.mrf.io/json/web?site_id=120&client_id=fec803ab-7426-4617-ab35-806db795b85c&canonical_url=https%3A%2F%2Fplayground.mrf.io%2Fsimulate%3FsiteUrl%3Dhttps%253A%252F%252Fplayground.marfeel.com%252Ftemplates%252Fbranded-content-article.html%26requestHostname%3Dplayground.marfeel.com%26experienceUrl%3Dcroupier&referrer=https%3A%2F%2Fplayground.marfeel.com&geo=__INJECT_GEO__&dynamic_flowcards=${encodeURIComponent(dynamic_flowcards)}`);
 
-			expect(isAtMinimizedSnapPoint).equal(true);
+			expect(croupier.cards.BrandedContentFlowcard.content.url).equal('https://flowcards.mrf.io/transformer/playground.marfeel.com?url=https%3A%2F%2Fplayground.marfeel%2Fexperiences%2Fbranded-content%2Fbranded-content-article.html&vars=currentUrl,https%3A%2F%2Fplayground.mrf.io%2Fsimulate%3FsiteUrl%3Dhttps%253A%252F%252Fplayground.marfeel.com%252Ftemplates%252Fbranded-content-article.html%26requestHostname%3Dplayground.marfeel.com%26experienceUrl%3Dcroupier,technology,0,flowcardId,FC_DC_UNKNOWN');
 		}
-	});
+	}
+}
+
+const suite = async function() {
+	const fixture = {
+		articleTitle: 'Branded Content example',
+		url: getUrlFixture({
+			siteUrl: 'https://playground.marfeel.com/templates/branded-content-article.html',
+			experienceUrl: 'croupier'
+		})
+	};
+
+	test.with(experience)
+		.test('croupier response matches the expected card', customCheckers.croupierMatches())
```

Comment:
I couldn't understand what was the idea behind pulling this experience from croupier to run the test, so I refactored it to check that croupier is actually returning what we expect. Not sure if it's worth to include a `deepEqual` function to check the full json response.